### PR TITLE
feat:product notes & edit qty

### PR DESCRIPTION
**Pull request by yohanavitrawati**
Screen Shot 2022-03-08 at 08 39 26
Screen Shot 2022-03-08 at 08 40 22
Screen Shot 2022-03-08 at 08 41 54
Screen Shot 2022-03-08 at 08 42 08
Screen Shot 2022-03-08 at 08 42 27
Screen Shot 2022-03-08 at 08 42 47
Screen Shot 2022-03-08 at 08 42 58
Screen Shot 2022-03-08 at 08 43 18
Screen Shot 2022-03-08 at 08 43 34
Screen Shot 2022-03-08 at 08 43 51
Screen Shot 2022-03-08 at 08 44 12

### CODE REVIEW

File: public/scss/components/OrderSummaryBox.module.scss [hunk 1]
```diff
@@ -543,18 +535,131 @@
...
     grid-template-areas:
-    "image title title"
-    "image total total"
-    "image total total" !important;
+    "input btnMin"
+    "input btnPlus";
+    border: 1px solid $color_light_border;
+    border-radius: 3px;
+
+    button{
```
Comment (virginiaritonga) [inline public/scss/components/OrderSummaryBox.module.scss:579]: bisa dienter aja mbaa

File: public/scss/components/OrderSummaryBox.module.scss [hunk 2]
```diff
@@ -543,18 +535,131 @@
...
+        grid-area: btnMin;
+        
+        &::before
+        {
+          background-image: url("data:image/svg+xml,%0A%3Csvg width='10' height='6' viewBox='0 0 10 6' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M4.99996 2.828L2.17196 5.657L0.756958 4.243L4.99996 0L9.24296 4.243L7.82796 5.657L4.99996 2.828Z' fill='%23333333'/%3E%3C/svg%3E%0A");
+        }
+      }
+    }
+
+    input[type=number] {
```
Comment (virginiaritonga) [inline public/scss/components/OrderSummaryBox.module.scss:630]: bisa dienter aja mbaa

File: public/scss/components/OrderSummaryBox.module.scss [hunk 3]
```diff
@@ -543,18 +535,131 @@
...
+      {
+        outline: 0;
+      }
+
+      /* Firefox */
+      -moz-appearance: textfield;
+
+      /* Chrome, Safari, Edge, Opera */
+      &::-webkit-outer-spin-button,
+      &::-webkit-inner-spin-button {
```
Comment (virginiaritonga) [inline public/scss/components/OrderSummaryBox.module.scss:649]: bisa dienter mbaa

**Review by virginiaritonga — CHANGES_REQUESTED** [review]

**Review by falmesino — APPROVED** [review]

**Review by virginiaritonga — APPROVED** [review]

**Review by virginiaritonga — APPROVED** [review]

File: components/OrderSummaryBox/index.tsx [hunk 4]
```diff
@@ -106,7 +109,19 @@ const OrderSummaryBox: FC<OrderSummaryBoxPropsType> = ({
   const classesOrderSum = () => {
     return {
       ...classesOrderSummary,
-      containerClassName: `${styles.ordersummary_container} ${page === "payment_method" ? "none" : ""}`
+      headerClassName: `${styles.ordersummary_header} ${page === "payment_method" ? "d-none" : ""}`
```
Comment (muhammadanova) [inline components/OrderSummaryBox/index.tsx:112]: Kayanya bisa gini
${page === "payment_method" && "d-none"}

Comment (muhammadanova) [reply components/OrderSummaryBox/index.tsx:112]: Kalo ternary else nya string kosong mending dibuat AND daripada option

**Review by muhammadanova — CHANGES_REQUESTED** [review]

**Review by muhammadanova — APPROVED** [review]

File: public/scss/components/CartDetails.module.scss [hunk 5]
```diff
@@ -391,4 +392,56 @@
...
+      padding: 10px 15px;
+      margin-bottom: 3px;
+      outline: none;
+    }
+  }
+
+  &_itemEditNote
+  {
+    display: inline-block;
+    width: 19px;
```
Comment (inezamandha) [inline public/scss/components/CartDetails.module.scss:433]: mba vivi yang ini apakah bisa pakai @include fixedSize?

**Review by inezamandha — CHANGES_REQUESTED** [review]

**Pull request merged**